src/components/Uploader.css

```diff
@@ -1,14 +1,14 @@
-@import "part:@sanity/base/theme/variables-style";
+@import 'part:@sanity/base/theme/variables-style';
 
-.root {
+/* .root {
   padding: var(--small-padding);
   border-radius: var(--input-border-radius);
   transition: all 0.1s linear;
   color: var(--input-color);
   background-color: var(--input-bg);
   box-shadow: 0 0 3px color(var(--gray) a(50%));
   min-height: 160px;
-}
+} */
```

Comment:
this commented code can probably be removed

Reply:
Very true, good catch! Removed in ef4b78f